app/controllers/parties_controller.rb

```diff
@@ -54,6 +54,7 @@ def members_letters
   def current_members_letters
     letter = params[:letter]
     party_id = params[:party_id]
+    #binding.pry
```

Comment:
We should try not to leave binding.pry in the code, even if it is commented out.